impl/client/src/main/javascript/web/pentaho/visual/models/i18n/model.properties

```diff
@@ -2,7 +2,7 @@
 
 ## Abstract Chart
 abstract.label=Abstract Chart
-abstract.description=
+#abstract.description=
```

Comment:
Well for the abstracts, I say they could be removed. I only left it there so that people know it exists.
But for the other locales I commented the ones missing translations.

I will remove the abstract ones